scripts/commit.sh

```
@@ -281,6 +314,128 @@ function commit_script {
+            fi
+            exit
+
+        elif [ "$choice" = "e" ] || [ "$choice" = "E" ]; then
+            echo -e "${YELLOW}Edit the AI generated message:${ENDCOLOR}"
+            if [ -n "${msg}" ]; then
+                # Create temp file with AI message
+                commitmsg_file=".commitmsg__"
+                echo "$ai_commit_message" > $commitmsg_file
+
+                while [ true ]; do
+                    $editor $commitmsg_file
```

Shellcheck (0.10.0)

[warning] editor is referenced but not assigned.

(SC2154)